docs/developer/eventing/sinks/README.md

```diff
@@ -1,29 +1,12 @@
 # About sinks
 
-When you create an event source, you can specify a _sink_ where events are sent to from the source. A sink is an Addressable resource that can receive incoming events from other resources. Knative Services, Channels, and Brokers are all examples of sinks.
+When you create an event source, you can specify a _sink_ where events are sent to from the source. A sink is an _Addressable_ or a _Callable_ resource that can receive incoming events from other resources. Knative Services, Channels, and Brokers are all examples of sinks.
```

Comment:
_Callable_ is intended to inherit from _Addressable_; every _Callable_ should be _Addressable_.

With that said, this seems fine.